superset/charts/commands/exceptions.py

```diff
@@ -30,7 +30,7 @@
 
 class TimeRangeUnclearError(ValidationError):
     """
-    Time range is in valid error.
+    Time range is unclear error.
```

Comment:
Not caused by this PR, but I think "Unclear" is slightly ambiguous. I think it could be more clear to call it `TimeRangeUnparseableError` for time ranges that are totally incorrect/unparseable, and have a separate `TimeRangeAmbiguousError` for time ranges that are ambiguous, e.g. '1 year' as this exception seems to be used now.

Btw, I noticed that there are unit tests for `parse_human_datetime` in the `test_parse_human_datetime` integration test, but the function already has `doctests` attached to it. If it's not a big deal, could we move all these tests into one place, either into the doctests or into the existing test function in `test_parse_human_datetime`?

Reply:
Nice! I have done these changes.